database/seedRecords.js

```diff
@@ -0,0 +1,94 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const headerTemplate = 'id,name,description,articleNumber,details,materials,sustainability,width,height,length,weight,packages,shortDesc,thread-count,Pillowcase quantity, Duvet cover length, Duvet cover width, Pillowcase length, Pillowcase width, fitting, imagesUrls\n'
+const writeUsers = fs.createWriteStream('database/records.csv');
+writeUsers.write(headerTemplate, 'utf8');
+
+function writeTenMillionUsers(writer, encoding, callback) {
+
+  let i = 10000000;
+  let id = 0;
+  function write() {
+    let ok = true;
+    do {
+      i -= 1;
+      id += 1;
+      const data = dataGenerator(id);
+      if (i === 0) {
+        writer.write(data, encoding, callback);
+      } else {
+// see if we should continue, or wait
+// don't pass the callback, because we're not done yet.
+        ok = writer.write(data, encoding);
+      }
+    } while (i > 0 && ok);
+    if (i > 0) {
+// had to stop early!
+// write some more once it drains
+      writer.once('drain', write);
+    }
+  }
+write()
+
+}
```

Comment:
I know where you got the inspiration for this block of code! I used the same and it served me well.